src/tensor_ops.jl

```diff
@@ -59,7 +66,7 @@ Computes the norm of a tensor
 """
 Computes the outer product between two tensors `t1` and `t2`. Can also be called via the infix operator `⊗`.
 """
-@inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
+# @inline otimes{order, dim}(t1::AbstractTensor{order, dim}, t2::AbstractTensor{order, dim}) = otimes(promote(t1, t2)...)
```

Comment:
I think this one is used then: https://github.com/KristofferC/ContMechTensors.jl/blob/bd4494c77c8fb31eeb6ad35e718c14c4e896b9f2/src/tensor_ops.jl#L64